ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,25 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysatSpaceWeather development.
+
+Institutions
+------------
+ - The Catholic University of America (CUA)
+ - Cosmic Studio
+ - Defense Advanced Research Projects Agency (DARPA) Defense Sciences Office
+ - National Aeronautics and Space Administration (NASA)
+ - National Science Foundation (NSF)
+ - Office of Naval Research (ONR)
+
+Programs
+--------
+ - NSF 125908, AGS-1651393
+ - Naval Research Laboratory N00173191G016 and N0017322P0744
+ - The Space Precipitation Impacts (SPI) project at Goddard Space Flight Center through the Heliophysics Internal Science Funding Model
+
+Disclaimers
+===========
+Any opinions, findings, and conclusions or recommendations expressed in this
```

Comment:
```suggestion
Any opinions or actions taken by the listed funding institutions are those of the institutions and do not necessarily reflect the views of the pysat development team or individual authors. Any opinions, findings, and conclusions or recommendations expressed in this
```

Reply:
@rstoneback please also update the wording on the ecosystem template, so it can propagate easily.